stage-config.json.asc

```diff
@@ -1,25 +1,25 @@
 -----BEGIN PGP MESSAGE-----
 
-hQEMAy5VCdmwb1c1AQf/a1pRFXPBDDYg8FS2NToA3KvTWt/Aio454g6lkXQSqycf
-PR81aNiFyhCQ6RIG+8QVrPVPoR2TlxiTD+Wrc0hMalMi/LLa3ROTlrwwSZi8xlEp
-7NcIAfyfkRlug7prxhh8W8SWYjSV1TyTGhUpco5poVdRIFBPMoGyxqb03FazlRrN
-H2zHpQDMLoeySu9nhDTpG9zR3Bf3IzuNKoU5dn6B5qFzsTiRudJ1LmlRiVp+ieFZ
-YNZnSO3XPGD1vdTbNdY/3R7ae9cdBS8ny8LBSAbXV+DezOfh373l+B4oE6fOUUTp
-QYs8xPrbINN4/stqmcKlNHv1jY+ISZ069Q3S39NVjNLpAZvVi22LSbQT7/rSjGIP
-ot2OvYashcr2gM86ebJCMTlkCskQE1MwKbv47T19UMFWw50mUwY/pBaqXnfFp/uq
-nNbaFpqDsdhZFPGY710XPX0m+VFbZMHQ0WuPtsoiyqSrNtvyev4jcM8szptRxeHE
-/NhXr06FDi/t32C50qah/sMVeDdO8RaueIAg20JFiPMq/aIIZJmGsETnJSUACTf2
-9Ikinpi47rpTEzL9+yhqanbrh4YA0b+/tfRAj6eLU3irUa5HrdYlqHLL1GHBJ6D2
-Qmgl2SeAsDx8GxFKuxWINoeErXXtL3H3xglFGCy84Fp69sbx1ThJhFwdDcwu/qwo
-87mpH/Ttekn6wYSwB0C84E/jme1ncy9o+tsgoPUaqcHaUzxAFMg6H2IL7MA0FNz0
-InwgFOvzCs33megvDS3v7ddpJeKdmq0iiNHiP/cpkmUCvu7k/xH7bvtRsRFlm6Fw
-KsrWKIwjA7hm4L/6O7wm8mP2DsPPR8d3c/OshWmPJAvtESf56vVGkejuFs15L0Dl
-oQypxzxptkmEX5E5KIYoIuDjXOE3sfdGPGLmPAZKf1fGv3P3ORF5/e2z1MG2xynK
-oWEkzgUyDn1PpbXk2xdP4pXGXhS8ZTfYXay6yIq9OTdVYoQMZVyWyPnNGpLsFBA9
-JOkdmUzyfk4ZzmubDk3TynbAD7YiywratOSMf6rsTE0mUnMI3zHefhIWH0YGIF6l
-SXiK3Wkn1z0EhNsZ78XXep/ARx2hWIHo8cETIlBflwlA975Z+0jrDYJwV5SHFk+9
-FhHLpPuTrbM1oRkLtHyO+SUk6Gjm6GOugTN2H5BrL4qywE7QcdtFOXGmAIHtyXSa
-s1YyC+KMSCYwR5fCE5yxkZ6Ahv776rLvxwVkRHkUj/dI5aF9u+oxBeTwgiXUT9kp
-h5WvFNbU+RETLhIWOMVXlwN6Uutdl6QGSyvPWQ+Suw0sow==
-=gxYj
+hQEMA4TG4yJX2eSjAQf/dxgHSoVto497kd+xC2vwTZn12f3Nljv8JgKBM/TCZmDF
+ANxzB9ziAVzAuT3Oi1vX98Jsz2FVEbpOgzLu0fcIRu+ST9I2j0awn9J3AN9BIuF2
+1PJzHuvDihMWqM0WCT7Q9DTOQk+z7PF8o/q6SXTQJ0mNavl1J08aYX+nrLBnjZdz
+E2SR+CVKBAPMFaq32ctsPHQdCVYPLxRLAB4EZv7G+bsEEHFlJ3c1hKcObLYNT9kM
+TawfkgFGq88gQhpfyVzOnkrkQpKazA0agk4HXE7rH/8UDGJhQkr2WsTjcxb+KuiQ
+P/jguDslK6RLyjt6mKZ6SBIlw47ewp3WA+tKX9s/xtLpAcch6a6dl8znT9n/HaTR
+ZuWv62xJn/IWnkfuO+EerG+1yL+iUEg0uyMkal9EfH6mX1Z03McJvF/j/q/VTvTn
+C9qRg/0Ru0KWU6TAUfPefsfLvVbkieLZ1QEQTXRwehHCf1U9mpowuGR8n1o4mxip
+1HS5bh8QkUJxnuMnWHm5VZkHAJxBDUItxud70SAdH2M4x/Zf+gLXnqCKWqp4UHz1
+GaRL31+xMuAcxpDW2v7tEK0lHfRPlzDWcQX9RABQRlLLopl1qbTlM7pOFHczVXXj
+n17hPqAtuz9ua/Qs8dSlxuuLCCPGKMYbtXRZUsxpKg0TX8Vo2XwkYUbyA5qaEKhB
+mIhK3zSMZj77QbMOD88cqazckGsXTqSBs/Lz/8VF42xnBWmxS50WOtA2PO6zPmJP
+VdPuze/2gdUKqGKVgVvzWTC3lxCMy32ksN1pwpd8dUyJdk4o8mPzjaakTU1JdPDp
+IGYZVGQ91fPQ0U8nPqm2Inw24g2WSLCRqUB+YeIFGXAp2Nyx11N/JnkrS37DU/Sv
+utqCvxmMBW/NA8Olv75GH2Bk2/jgnrBAFzjsPv4RT13g/mDB6TcQSvC1sGbMzpdm
+Xw14M4QZ7Pf22meunOJ6sHTZ4ivKz5rUIt4lQiARGJxDfU1Z3B9/njS8f146FVCt
+Gu7w5qJ929LOS5s6CCr1zqfADWZ9TzrlQePyzVphQ/X2hO4J7W2bWT21hQUlt6BG
+LnFInztpB/l88rkdu/dIBFO5ss3Nyw26V8N64Eb+nSSmbxdYkY7/Bw+i5604v2ak
+KQaDUoy2J9N2iuToPQNDC6FN82U6IUgpwCRm75CH/Y942BcatGx4RREakiNU4ayx
+jeR9YESa2rkWAEUN//aGyKeUs82EUHQiRmPfHg3mtzbgPvDyj5k9AqcTiJC0bNc0
+Zjqkf7xdOnBeoRI4TRiR8NzrX6TUd8dNT34CQypSpAQ=
+=1V4u
```

Comment:
@rpappalax: Same w/ this file... [@karlht's version](https://github.com/mozilla-services/sync-tps-setup/blob/master/stage-config.json.asc) hasn't been modified in ~2-3 months. Not sure if we did something funky locally, but we should probably back out this file as well.